src/users/users-thunk.js

```diff
@@ -37,4 +46,9 @@ export const findAllUsersThunk = createAsyncThunk(
 export const createUserThunk = createAsyncThunk(
     'createUser',
     async () => await createUser()
+)
+
+export const updateThunk = createAsyncThunk(
+    'updateUser',
+    async() => await updateUser()
```

Comment:
needs to pass in required parameters here

Reply:
also, when the updateThunk is fulfilled, currentUser should be set to the updated user returned by the response

Reply:
can accomplish this by looking at the meta property of the action object (action.meta)